_includes/footer.html

```diff
@@ -1,27 +1,39 @@
 <div id="footer">
-    <div>
-        <a href="{{site.social.facebook}}">
-            <span class="fa-stack fa-lg">
-                <i class="fa fa-circle fa-stack-2x"></i>
-                <i class="fa fa-facebook fa-stack-1x fa-inverse"></i>
-            </span>
-        </a>
-        <a href="{{site.social.instagram}}">
-            <span class="fa-stack fa-lg">
-                <i class="fa fa-circle fa-stack-2x"></i>
-                <i class="fa fa-instagram fa-stack-1x fa-inverse"></i>
-            </span>
-        </a>
-        <a href="mailto:{{site.contact.email}}">
-            <span class="fa-stack fa-lg">
-                <i class="fa fa-circle fa-stack-2x"></i>
-                <i class="fa fa-envelope fa-stack-1x fa-inverse"></i>
-            </span>
-        </a>
+    <div id="footer-contact">
+        <div id="footer-connect">
+            <h2>Connect with us:</h2>
+            <p>info@bitbybitcoding.org</p>
```

Comment:
nitpick, but could you add the variable back here? (the `{{ site.contact.email }}` thing).  this is a jekyll variable that contains our email address so that if we ever have to change it, we only need to change it in one place, rather than having to remember and fix every file where it is used

Reply:
Definitely, that all makes sense.